app/controllers/general.php

```diff
@@ -746,7 +749,16 @@ function router(App $utopia, Database $dbForPlatform, callable $getProjectDB, Sw
     ->inject('logger')
     ->inject('log')
     ->inject('queueForUsage')
-    ->action(function (Throwable $error, App $utopia, Request $request, Response $response, Document $project, ?Logger $logger, Log $log, Usage $queueForUsage) {
+    ->inject('connectionForProject')
+    ->action(function (Throwable $error, App $utopia, Request $request, Response $response, Document $project, ?Logger $logger, Log $log, Usage $queueForUsage, Connection $connectionForProject) {
+        if (
```

Comment:
for anything uncaught by this, we should also have a similar check in the error handler for the http server here https://github.com/appwrite/appwrite/blob/1.6.x/app/http.php#L348

Reply:
Added the check there too